presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -6197,7 +6197,7 @@ public void testMultipleOrderingOnSameCanonicalVariables()
     }
 
     @Test
-    public void tesMultipleConcat()
+    public void testMultipleConcat()
```

Comment:
I'd prefer to fix this in a separate PR in case it breaks a subclass somewhere, including ones not in this repo

Reply:
Removed it and opened https://github.com/prestodb/presto/pull/23224